### PR TITLE
TASK: Add a CLI command to index a node by identifier and workspace name

### PR DESCRIPTION
**Pull request by dfeyer**
This change introduce a new CLI command to index a single node in the current index:

    flow nodeindex:indexnode --identifier c480b96f-ee9c-b304-3feb-570182a08236

### CODE REVIEW

File: Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Command/NodeIndexCommandController.php [hunk 1]
```diff
@@ -126,6 +129,62 @@ public function showMappingCommand()
...
+            $this->outputLine('  node type: %s', [
+                $node->getNodeType()->getName()
+            ]);
+            $this->outputLine('  dimensions: %s', [
+                json_encode($dimensions)
+            ]);
+            $this->nodeIndexer->indexNode($node);
+        };
+
+        $indexInWorkspacce = function ($identifier, Workspace $workspace) use ($indexNode) {
```
Comment (aertmann) [inline Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Command/NodeIndexCommandController.php]: spa**cc**e

File: Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Command/NodeIndexCommandController.php [hunk 2]
```diff
@@ -126,6 +129,62 @@ public function showMappingCommand()
     }
 
     /**
+     * Index a single node by the given identifier and workspace name
+     *
+     * @param string $identifier
+     * @param string $workspace
```
Comment (aertmann) [inline Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Command/NodeIndexCommandController.php]: @return void

**Comment by aertmann** [comment]
useful command, left some nitpicks

**Comment by dfeyer** [comment]
@aertmann Thanks, at some point if I send you a tshirt with "@return void" in you back, don't mind too much ;)

**Comment by aertmann** [comment]
@dfeyer hahah feel free, although actually don't really care about that rule, just have a habit of looking for it and many people seem to forget it all the time 😄

**Pull request merged**